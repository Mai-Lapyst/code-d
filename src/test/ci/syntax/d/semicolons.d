foo();
foo(); // comment
foo(); /* comment */
foo();;
foo(); bar();