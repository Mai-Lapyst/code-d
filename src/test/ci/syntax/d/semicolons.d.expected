[{"start":0,"end":3,"scope":"entity.name.function.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"meta.statement.end.d"}]
[{"start":0,"end":3,"scope":"entity.name.function.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"meta.statement.end.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":17,"scope":"comment.line.d"}]
[{"start":0,"end":3,"scope":"entity.name.function.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"meta.statement.end.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":9,"scope":"comment.block.begin.d"},{"start":9,"end":18,"scope":"comment.block.content.d"},{"start":18,"end":20,"scope":"comment.block.end.d"}]
[{"start":0,"end":3,"scope":"entity.name.function.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"meta.statement.end.d"},{"start":6,"end":7,"scope":"meta.statement.end.d"}]
[{"start":0,"end":3,"scope":"entity.name.function.d"},{"start":3,"end":4,"scope":"source.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"meta.statement.end.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":10,"scope":"entity.name.function.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":12,"scope":"source.d"},{"start":12,"end":13,"scope":"meta.statement.end.d"}]